templates/admin/asset/image_editor.html.twig

```diff
@@ -127,13 +142,47 @@
             return false;
         });
     }
+
+    function manipulateMenuBar() {
+        const menuBar = document.getElementsByClassName('menu_bar')[0];
+        const liElement = document.createElement('li');
+        const aElement = document.createElement('a');
+        const spanElement = document.createElement('span');
+
+        aElement.setAttribute('id', 'save_button');
+        spanElement.setAttribute('class', 'name trn');
+        spanElement.innerText = parent.t('save');
+
+        aElement.appendChild(spanElement);
+        liElement.appendChild(aElement);
+        menuBar.prepend(liElement);
+
+        document.getElementById('main_menu_0_0').addEventListener('click', () => {
+            document.getElementById('main_menu_1_10').remove();
+            document.getElementById('main_menu_1_11').remove();
+        })
+
+        removeHref();
+
+        document.getElementsByTagName('nav')[0].addEventListener('click', (e) => {
+            removeHref();
+        });
+    }
+
+    function removeHref(className = 'main_menu') {
+        const allLinkElements = document.getElementsByTagName('nav')[0].querySelectorAll("a");
+        for (let a of allLinkElements) {
+            if (a.getAttribute('href') === 'javascript:void(0)') {
+                a.removeAttribute('href');
```

Comment:
Let's add a line like this here:

`a.classList.add('cursor-pointer');`

and some CSS

```CSS
.cursor-pointer {
    cursor: pointer;
}
```
As we removed the href the styling of the mouse cursor is now a bit broken (default mouse cursor instead of pointer).

Reply:
Thanks, it works - we need the `.cursor-pointer` class also on the save button :)